### DISCUSSION TITLE
Crash after autolaunch INAV 4.1

### DISCUSSION THREAD

**Discussion by zoran-grujic** [iNavFlight/inav#7999] [General]
Model is:
Wing, no tail, Phantom FX-61
MATEK F765-Wing
INAV 4.1
[LOG00171.TXT](https://github.com/iNavFlight/inav/files/8599858/LOG00171.TXT)

After catapult launch it gained altitude up to 80 m as it should, and then went directly down. After autolaunch it went into angle althold mode - like was set.  It was terrible impact. Lot of damage. LiPo battery smashed - no fire I was lucky. 

Possible issues:
Wrong CG
Failed servo
Bad settings after upgrade from 4.0
Software issue - heading data looks erratic

Any suggestions?

**Comment by stronnag** [comment]
The profile of the climb is strange, with the two steep sections.  I expect it stalled at the apex (speed below 4 m/s). The darker points are launch, the lighter cyan angle / althold.

![Screenshot from 2022-05-04 15-25-06](https://user-images.githubusercontent.com/158229/166702874-11c564a3-e6ae-405f-800a-0114285e9008.png)

**Reply by zoran-grujic** [reply]
Can this happen if the plane was too nose heavy? It is also unexpected that it could not recover from the stall from 80 m high point. You are right the climb angle was strange, first too large, then sort of getting too flat. Different from my normal launch...
